backend/src/Core/Secrets.cs

```diff
@@ -58,9 +60,14 @@ private static void ValidateSecrets(string volume, Secrets sec)
                 throw new Exception($"Unable to read secrets from volume: {volume}");
             }
 
-            if (string.IsNullOrWhiteSpace(sec.CosmosCollection))
+            if (string.IsNullOrWhiteSpace(sec.CosmosCollection))
+            {
+                throw new Exception($"CosmosCollection cannot be empty");
+            }
+
+            if (string.IsNullOrWhiteSpace(sec.CosmosLease))
             {
-                throw new Exception($"CosmosCollection cannot be empty");
+                throw new Exception($"CosmosLease cannot be empty");
```

Comment:
Let's not put the lease in the files where we're storing secret/sensitive data.  The lease is an identifier for this changefeed processor. For the moment, let's just use a constant value for the LEASE_NAME.  As we will want any and all API instances to get the same notifications.